platform/persistence/platform-persistence-core-impl/src/main/java/org/codice/ddf/persistence/internal/PersistentStoreImpl.java

```diff
@@ -61,38 +63,43 @@ public PersistentStoreImpl(SolrClientFactoryImpl clientFactory) {
     }
 
     @Override
-    // Input Map is expected to have the suffixes on the key names
-    public void add(String type, Map<String, Object> properties) throws PersistenceException {
+    public void add(String type, Collection<Map<String, Object>> items) throws PersistenceException {
         LOGGER.debug("type = {}", type);
         if (type == null || type.isEmpty()) {
             throw new PersistenceException(
                     "The type of object(s) to retrieve must be non-null and not blank, e.g., notification, metacard, etc.");
         }
-        if (properties == null || properties.isEmpty() || properties.containsValue("guest")) {
+        if (items == null || items.isEmpty()) {
```

Comment:
✏️  Should be able to do the same .isEmpty check here as you're doing below so you don't have to do the null check